pkg/operator/controller/gatewayclass/controller.go

```diff
@@ -131,17 +131,19 @@ func (r *reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 	if _, _, err := r.ensureServiceMeshOperatorSubscription(ctx); err != nil {
 		errs = append(errs, fmt.Errorf("failed to ensure ServiceMeshOperatorSubscription: %w", err))
 	}
-	if _, _, err := r.ensureServiceMeshControlPlane(ctx, &gatewayclass); err != nil {
+	if _, _, err := r.ensureIstio(ctx, &gatewayclass); err != nil {
 		errs = append(errs, err)
 	} else {
-		// The OSSM operator installs the maistra.io CRDs, Need to create the SMCP watch after the OSSM operator is installed.
-		// Using sync.Once here, to start the watch for SMCP, which should run only once.
-		r.startSMCPWatch.Do(func() {
-			isOurSMCP := predicate.NewPredicateFuncs(func(o client.Object) bool {
-				return o.GetName() == operatorcontroller.ServiceMeshControlPlaneName(r.config.OperandNamespace).Name
+		// The Sail operator installs the istios.sailoperator.io CRD.
```

Comment:
Change to reference OSSM3 instead of Sail?

Reply:
I don't think it matters for code comments.  As developers, we know that the OSSM operator is the Sail Operator.  I changed it for consistency, though: https://github.com/openshift/cluster-ingress-operator/compare/30b576c4293b2962b137bcb93ff77f278d7bf681..26a3918c7374cc16eac04b87c9e4267591c304a5